app/components/ErrorBoundary.tsx

```diff
@@ -21,7 +21,7 @@ export function ErrorBoundary() {
 				</CardTitle>
 			</CardHeader>
 			<CardContent className="text-center space-y-4 border-0 bg-transparent">
-				<p className="text-muted-foreground">
+				<p className="">
```

Comment:
good less is more. But we can remove empty className

Reply:
Done